get.sh

```diff
@@ -626,40 +628,55 @@ fi
 checkRepoSHA()
 {
 	output_file="$TESTDIR/TKG/SHA.txt"
-	echo "$TESTDIR/TKG/scripts/getSHA.sh --repo_dir $1 --output_file $output_file"
-	$TESTDIR/TKG/scripts/getSHA.sh --repo_dir $1 --output_file $output_file
+	if [ -e ${output_file} ]; then
+		echo "rm $output_file"
+		rm ${output_file}
+	fi
+
+	sha_file="$TESTDIR/TKG/SHA.txt"
```

Comment:
We do not need another variable `sha_file`. It is the same as `output_file`.

Please do not change the logic of generating SHA.txt. We will need to use SHA.txt to validate the changes for testenv.proeperties.